models/Budget.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+//Create Schema
+const budgetSchema = Schema({
```

Comment:
@RobWawrzak   -- Can you please fix the schema for my budget section:
      Name:  required
      Amount:  required
      Frequency: required
      Date: required
      Category: required
**See my minimum JSON in trello -- https://trello.com/c/mtPIVtl4/41-final-minimum-json-required#comment-5be223cfcb51481615af6b39